avm/res/databricks/workspace/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -112,7 +113,7 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
       keyVaultResourceId: nestedDependencies.outputs.keyVaultDiskResourceId
       rotationToLatestKeyVersionEnabled: true
     }
-    storageAccountName: 'sa${namePrefix}${serviceShort}${substring(uniqueString(baseTime), 0, 3)}'
+    storageAccountName: 'sa${namePrefix}${serviceShort}001'
```

Comment:
I can't help but wonder if there was an original purpose for the 'randomness'. Any idea why it was implemented that way?

Reply:
It was for testing, waaf and max with same dependencies had different deployment errors which forced me to rule out and test this case with an additional random string. In connection with the random location deployments it again has different depoyment outcomes. I found a stable way and should be good now.